3570/3/rating_places/db/seeds.rb

```diff
@@ -0,0 +1,52 @@
+places = [
+  { name: 'Друзья',
+    short_description: 'Один из крупнейших ресторанов в СНГ',
+    description: 'Каждый день здесь отдыхает до 2000 гостей, выпивается более'\
+    'чем 500 литров пива, прослушивается около 4 часов живой музыки.'\
+    '«Друзья» – это 5 000 кв. метров дружественной атмосферы, высококлассного'\
+    'сервиса, качественной музыки и хорошего настроения!',
+    latitude: 53.927257,
+    longitude: 27.565514 },
+  { name: 'Высота 281',
+    short_description: 'Винный бар в самом сердце Минска.',
+    description: 'Отдельный кальянный зал, летняя терраса на двух уровнях'\
+    'и самый аппетитный вид на город.Завтраки с открытия и до 16:00, авторские'\
+    'рецепты c популярными блюдами белорусской кухни на основе локальных продуктов.'\
+    'У заведений бывают понятные с первого раза названия, а бывают и не совсем с'\
+    'первого, и не совсем понятные.Так вот 281 метр — это расположение Минска над'\
+    'уровнем моря. Именно с этой высоты, с «балкона» заведения и его панорамных'\
+    'окон открывается чудесный вид на столицу с трех сторон. Здесь можно вкусно'\
+    'поесть, отдохнуть и насладиться тем городом, в котором ты здесь и сейчас.'\
+    '2-хуровненный городской бар будет интересен ценителям кальянов и тихих вечеров'\
+    'и каждому, кто не прочь встретить закат с бокалом вина.',
+    latitude:  53.9042,
+    longitude: 27.5554 }
+
+]
+
+comments = [
+  { text: 'Отзыв номер раз', rating: 5, user_id: 1, place_id: 1 },
+  { text: 'Отзыв номер два', rating: 4, user_id: 2, place_id: 1 },
+  { text: 'Отзыв номер три', rating: 2, user_id: 3, place_id: 1 },
+  { text: 'Отзыв номер четыре', rating: 5, user_id: 1, place_id: 2 },
+  { text: 'Отзыв номер пять', rating: 1, user_id: 4, place_id: 2 }
+]
+
+users = [
+  { name: 'Ivan', email: 'ivanov@tut.by', password: '11111' },
+  { name: 'Petr', email: 'petrov@tut.by', password: '22222' },
+  { name: 'Dmitry', email: 'dmitriew@tut.by', password: '33333' },
+  { name: 'Semen', email: 'semenow@tut.by', password: '4444' }
+]
+
+places.each do |place|
+  Place.create(place)
+end
+
+comments.each do |comment|
```

Comment:
How you can create a comment without any user?

Reply:
@DmitryAvramec , he/she didn't add foreign keys. After adding the seed script will break down.